### ISSUE TITLE
Crashing report - Testnet 3.3 - Docker - Ducca#5707

### ISSUE THREAD

**Issue by icohigh** [CodaProtocol/coda#5985]
**Launched Block Producer and Snark Worker in docker.**
VPS with 8 cores 16 RAM. 

1. The first error is that it cannot download the archive:
```
[Info] Could not download genesis ledger from $uri: $error
	uri: "https://s3-us-west-2.amazonaws.com/snark-keys.o1test.net/genesis_ledger_accounts_e7fd308899bca6956b093b635988ea06630adbf540e13d38dcd2f7ab629da53c_f094d0eb814dbbff2f41f3f69c414331566f2c4a884c309944899ea59f101b9a.tar.gz"
	error: "(monitor.ml.Error\n (\"Process.run failed\"\n  ((prog curl)\n   (args\n    (--fail -o\n     /tmp/s3_cache_dir/genesis_ledger_accounts_e7fd308899bca6956b093b635988ea06630adbf540e13d38dcd2f7ab629da53c_f094d0eb814dbbff2f41f3f69c414331566f2c4a884c309944899ea59f101b9a.tar.gz\n     https://s3-us-west-2.amazonaws.com/snark-keys.o1test.net/genesis_ledger_accounts_e7fd308899bca6956b093b635988ea06630adbf540e13d38dcd2f7ab629da53c_f094d0eb814dbbff2f41f3f69c414331566f2c4a884c309944899ea59f101b9a.tar.gz))\n   (exit_status (Exit_non_zero 22)) (stdout \"\")\n   (stderr\n    (\"  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\"\n     \"                                 Dload  Upload   Total   Spent    Left  Speed\"\n     \"\\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\"\n     \"curl: (22) The requested URL returned error: 404 Not Found\" \"\"))))\n (\"Raised at file \\\"src/error.ml\\\" (inlined), line 9, characters 14-30\"\n  \"Called from file \\\"src/or_error.ml\\\", line 72, characters 17-32\"\n  \"Called from file \\\"src/deferred1.ml\\\", line 17, characters 40-45\"\n  \"Called from file \\\"src/job_queue.ml\\\" (inlined), line 131, characters 2-5\"\n  \"Called from file \\\"src/job_queue.ml\\\", line 171, characters 6-47\"))"
```
2. And Crash 2 hours after starting:
```
2020-09-17 09:26:43 UTC [Error] Error verifying transaction snark: $error
        error: "((rpc_error (Connection_closed (\"EOF or connection closed\")))\n (connection_description (\"Client connected via TCP\" (7aea31cfa2b4 38443)))\n (rpc_tag rpc_parallel_plain_0) (rpc_version 0))"
2020-09-17 09:26:43 UTC [Error] Error verifying transaction snark: $error
        error: "((rpc_error (Connection_closed (\"EOF or connection closed\")))\n (connection_description (\"Client connected via TCP\" (7aea31cfa2b4 38443)))\n (rpc_tag rpc_parallel_plain_0) (rpc_version 0))"
2020-09-17 09:26:46 UTC [Info] starting refreshing cpl 0 with key CIQAAA44GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA (routing table size was 0)
2020-09-17 09:26:46 UTC [Info] failed when refreshing routing table
+ echo 'Coda process exited with status code 99'
```

Logs for 2 hour:
[crash.log](https://github.com/CodaProtocol/coda/files/5237987/crash.log)

**Comment by icohigh** [comment]
**Snark Worker crashed every 5 minutes:**
`error: "((rpc_error (Connection_closed (\"EOF or connection closed\")))\n (connection_description (\"Client connected via TCP\" (07176ec01a6e 39551)))\n (rpc_tag rpc_parallel_plain_0) (rpc_version 0))"`

**Comment by emberian** [comment]
I believe this was fixed by #6036
